pkg/ecdsa/tss/protocol_announce.go

```diff
@@ -81,6 +101,18 @@ func AnnounceProtocol(
 
 	switch ctx.Err() {
 	case context.DeadlineExceeded:
+		for _, member := range keepMemberAddresses {
+			_, announcedPresence := receivedMemberIDs[member]
+			if !announcedPresence {
```

Comment:
Might be simplified a bit:
```
if _, ok := receivedMemberIDs[member]; !ok {
	logger.Errorf(
		"member [%v] has not announced its presence for keep [%v]; "+
			"check if keep client for that operator is active and "+
			"connected",
		member,
		keepAddress,
	)
}
```

Reply:
Done in https://github.com/keep-network/keep-ecdsa/pull/654/commits/e410b4d7c974b8d302c1e77cfea484636da18b37